src/main/java/com/github/dockerjava/jaxrs/filter/ResponseStatusExceptionFilter.java

```diff
@@ -31,6 +33,8 @@
  */
 public class ResponseStatusExceptionFilter implements ClientResponseFilter {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResponseStatusExceptionFilter.class.getName());
```

Comment:
without `.getName()` just class is enough

Reply:
Done